src/translations/da_DK.json

```diff
@@ -146,14 +146,14 @@
   "FOREVER_DOWNLOAD_PARAGRAPH": "Du er nu Hedvig-medlem. Download appen for at komme i gang, og husk at invitere dine venner!",
   "FOREVER_DOWNLOAD_DOWNLOAD_APP_CTA": "Download appen",
   "WEB_GENERIC_CAMPAIGN_ADDEDPERK": "Rabatkode fra {CAMPAIGN_NAME} er tilføjet.",
-  "CHECKOUT_SSN_LABEL": "",
```

Comment:
This is used in this pr and had to update the translation, so it just display "CHECKOUT_SSN_LABEL" but I guess will be updated in another pr